package-system/OpenXR/build_config.json

```diff
@@ -0,0 +1,65 @@
+{
+   "git_url":"https://github.com/KhronosGroup/OpenXR-SDK.git",
+   "git_tag":"release-1.0.22",
+   "package_name":"OpenXR",
+   "package_version":"1.0.22-rev1",
+   "package_url":"https://www.khronos.org/openxr/",
+   "package_license":"Apache-2.0",
+   "package_license_file":"LICENSE",
+   "cmake_find_source":"FindOpenXR.cmake",
+   "cmake_find_target":"FindOpenXR.cmake",
+   "cmake_build_args_common" : [ 
+      "--parallel"
+   ],
+   "cmake_generate_args_common" : [
+      "-DCMAKE_CXX_STANDARD=17",
+      "-DDYNAMIC_LOADER=OFF",
+      "-DOPENXR_DEBUG_POSTFIX=d"
+   ],
+   "cmake_install_filter":[
+      "*.h",
+      "*.lib",
+      "*.a"
+   ],
+   "build_configs": [
+      "Release",
+      "Debug"
```

Comment:
Are we still building debug builds for non-windows platforms? I thought we want to only have a debug configuration for windows only, and the rest only build the release version of the library?

Reply:
Mixing debug and non-debug is an issue with Windows for sure (for C++ libs). Yeah, there wasn't anything official about this, this came from a discussion in the 3PP slack channel. We didn't want to go back and change existing packages, but the consensus was that there was no real value in having debug 3P libraries unless they were necessary, which in the case of windows and c++ libraries they were (because of the dependence on windows release/debug c-runtime dll)

Reply:
I see. Thanks for letting me know. Since this package is already created and tested I rather move on, keep it simple and same for all platforms. But I'll have it in mind the next time I create another 3rdParty package.